src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -13,52 +13,72 @@ import sbt.Keys._
 import sbt._
 import scoverage.ScoverageKeys._
 
+// scalastyle:off method.length
```

Comment:
Propose to change this from 50 to 80. Will remove unneeded `scalastyle:off` configurations.